processes/release_processes/firecloud-develop.dot

```diff
@@ -49,17 +49,22 @@ digraph {
 
   coordinate_release_timing [shape=oval label="If downtime will be required, co-ordinate with the release engineer\nand user support/comms about timings and banners"];
 
+  update_release_ticket [shape=oval label="Update the release ticket to add a link to the release notes"];
```

Comment:
Historically, for some reason (a separate issue to be addressed elsewhere?) the GitHub release notes do NOT contain all of the Jira changes, thus may be insufficient for auditing purposes.

For the 55 release ticket, I had to run [generate my own diff](https://github.com/broadinstitute/cromwell/compare) to generate a fuller list of Jira tickets and issues that skipped auditing and weren't tracked in Jira.